src/compiler/printer.ts

```diff
@@ -528,12 +528,16 @@ const _super = (function (geti, seti) {
                     // JSX (non-expression)

```

Comment:
Minor: should this comment be updated? Since the SyntaxKind that it will print is `JsxExpression`